apps/web_fe/models.py

```diff
@@ -32,10 +32,15 @@ class ESGFNode(models.Model):
     node_data = JSONField()
 
     def refresh(self):
-        r = requests.get(
-            'http://' + self.host + '/esgf-node-manager/registration.xml', timeout=1)
+        status = True
+        try:
+            r = requests.get('http://' + self.host + '/esgf-node-manager/registration.xml', timeout=1)
+            if r.status == 200:
+                status = False
+        except Exception:
+            print 'http://' + self.host + '/esgf-node-manager/registration.xml'
 
-        if r.status_code != 200:
+        if status:
```

Comment:
Mind switching the polarity on that for the sake of clarity? Default status to False, set to True if status is 200, check `if not status` on the if statement?